portal-api/src/v1/gef/models/facilities.js

```diff
@@ -150,9 +150,9 @@ class Facility {
         this.submittedAsIssuedDate = req.submittedAsIssuedDate;
       }
 
-      // nullify values based on previous questions
+      // sets to now date to now
```

Comment:
sorry if it seems pedantic, but could we say something like "setting to midnight, we only care about the day". Just so it's totally clear for any future devs

Reply:
done